contracts/test/state/CommitmentAccumulator.t.sol

```diff
@@ -134,17 +134,6 @@ contract CommitmentAccumulatorTest is Test, MerkleTreeExample {
         });
     }
 
-    function test_verifyMerkleProof_reverts_on_wrong_path_length() public {
```

Comment:
Should we add a test that ensures that `verifyMerkleProof` now allows paths of different length?